app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -212,16 +214,16 @@ class BrowserToolbarIntegration(
         observerSecurityIndicatorChanges()
         if (store.state.findCustomTabOrSelectedTab(customTabId)?.isCustomTab() == false) {
             setBrowserActionButtons()
-            if (Features.SHOW_ERASE_CFR) {
-                observeEraseCfr()
-            }
+            observeEraseCfr()
         }
+        observeTrackingProtectionCfr()
```

Comment:
Just to double check seeing the above evaluation of `isCustomTab()`.
This CFR should be shown for custom tabs also and once shown there it's okay to not be shown again in normal browsing?